hieradata/common.yaml

```diff
@@ -194,7 +194,8 @@ profile::jenkinscontroller::jcasc:
   agents_setup:
     windows:
       agentDir: 'C:/Jenkins'
-      tempDir: 'C:/temp'
+      # Double Backslash is required (for EC2 plugin as we hackishly use the unix launched for Windows to use OpenSSH)
+      tempDir: 'C:\\\\Temp'
```

Comment:
For info @smerle33: with only 2 backslashes, you got the string `C:\Temp` because JCasc interpolates a double backslash into a single. So I've set it to 4 backslashes.

Tested locally